doc/rtd/topics/datasources/configdrive.rst

```diff
@@ -123,9 +123,9 @@ This provides cloud-init user-data. See :ref:`examples <yaml_examples>` for
 what all can be present here.
 
 .. _OpenStack: http://www.openstack.org/
-.. _introduction: http://docs.openstack.org/trunk/openstack-compute/admin/content/config-drive.html
+.. _metadata introduction: https://docs.openstack.org/nova/latest/user/metadata.html
```

Comment:
We could even link directly to the config drive section https://docs.openstack.org/nova/latest/user/metadata.html#config-drives
I meant to put that in the bug report, but had a copy-paste fail :D

Reply:
ok after one brief mistake, I think I updated it to show that URL. Can you verify these still look good? Thanks again!